src/CMakeLists.txt

```diff
@@ -141,32 +141,6 @@ set(keepassx_SOURCES_MAINEXE
     main.cpp
 )
 
-set(keepassx_FORMS
```

Comment:
Shouldn't these be added to the keepassx_core target or does the compiler find these automatically?